src/state/apis/nft/nftApi.ts

```diff
@@ -28,25 +29,38 @@ export const nftApi = createApi({
   endpoints: build => ({
     getNftUserTokens: build.query<NftItem[], GetNftUserTokensInput>({
       queryFn: async ({ accountIds }, { dispatch }) => {
-        const sources = [
-          zapperApi.endpoints.getZapperNftUserTokens,
-          covalentApi.endpoints.getCovalentNftUserTokens,
+        const services = [
+          getAlchemyNftData,
+          (accountIds: AccountId[]) =>
+            dispatch(zapperApi.endpoints.getZapperNftUserTokens.initiate({ accountIds })),
+          (accountIds: AccountId[]) =>
+            dispatch(covalentApi.endpoints.getCovalentNftUserTokens.initiate({ accountIds })),
         ]
 
-        const results = await Promise.all(
-          sources.map(source => dispatch(source.initiate({ accountIds }))),
-        )
+        const results = await Promise.all(services.map(service => service(accountIds)))
+
         const data = results.reduce<NftItem[]>((acc, result) => {
```

Comment:
probably better performance working with and object and then just returning `Object.values` instead

Reply:
https://github.com/shapeshift/web/pull/4582/commits/0a1a47df87ca344cbf511e3397422a2c94431a39 / https://github.com/shapeshift/web/pull/4582/commits/4906bdf620f10650dd85c8ef2e62cbf5698f07cf